controller/notify.go

```diff
@@ -28,8 +38,12 @@ func NewNotifyController(service *goa.Service, cRegistry collector.Registry, tRe
 
 // Send runs the send action.
 func (c *NotifyController) Send(ctx *app.SendNotifyContext) error {
-	nID := ctx.Payload.Data.Attributes.ID
 	nType := ctx.Payload.Data.Attributes.Type
+	if !checkAllowedSrvcAcc(ctx, nType) {
+		return jsonapi.JSONErrorResponse(ctx, errors.NewUnauthorizedError(fmt.Sprintf("Wrong token for '%s' notification type", nType)))
```

Comment:
Add a controller test to validate a 401 for this specific nType when service account name doesn't match/exist?